local_lib_pipinstal.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+# 202001108
+
+venv/bin/python -m pip install -e local-lib/test
```

Comment:
We don't use pip to manage our project, since @azlux has his own auto-update engine. I think it would be messy if we choose to use pip to manage the lib install...

Reply:
I was unaware of this (auto update disabled by default).